src/battle_ai_main.c

```diff
@@ -1324,6 +1324,7 @@ static s32 AI_CheckBadMove(u32 battlerAtk, u32 battlerDef, u32 move, s32 score)
             break;
         case EFFECT_EVASION_DOWN:
         case EFFECT_EVASION_DOWN_2:
+        case EFFECT_SPICY_EXTRACT:
```

Comment:
Why is this checking for evasion?

Reply:
Good question. I made a dedicated check for it in `AI_CheckBadMove`